src/units/store.rs

```diff
@@ -1,6 +1,5 @@
 use super::*;
 
-// TODO: need to make sure we never accept units of round > MAX_ROUND
```

Comment:
Is it outdated? Or do we ensure that somewhere?

Reply:
We ensure that in unit validation in `if full_unit.round() > self.store.limit_per_node()` ... 
At some point we will need to make sure honest nodes never exceed this limit, for that I have some ideas, but it's a different issue :)